tests/auth/mocmozillavpn.cpp

```diff
@@ -146,3 +146,5 @@ void MozillaVPN::abortAuthentication() {}
 void MozillaVPN::appReviewRequested() {}
 
 bool MozillaVPN::validateUserDNS(const QString&) const { return false; }
+
+void MozillaVPN::reset(bool forceInitialState) {}
```

Comment:
remove the param name, or add `Q_UNUSED(forceInitializeState)`. Same in the other mocmozilla.cpp